### PR TITLE
Stop duplicating inputs: Documentation

### PR DESCRIPTION
**Pull request by aofarrel**
Changes several docs around the twice-localized workaround. I also updated the issue itself: #2

### CODE REVIEW

File: _documentation_/for developers/cwl-vs-wdl-dev.md [hunk 1]
```diff
@@ -30,12 +28,11 @@ The second significant difference: The CWL technically has duplicated outputs. T
...
 
 ## ld-pruning.wdl
-* The twice-localized workaround is used for the same reason as it is in unique_variant_ids; it is taking in an input array of files from more than one input directory.
 * WDL does not have an equivalent to ScatterMethod:DotProduct so it instead scatters using zip().
 * check_merged_gds uses the chromosome file workaround.
 
 ## vcf-to-gds.wdl     
-* The twice-localized workaround is used in unique_variant_ids due to the fact the R script requires an array of GDS files and uses a config file which can only support an array of GDS files if each GDS file is in the same parent directory. As the GDS files are generated from a scattered task, they are sometimes given unique input directories on some file systems. As such, they first need to be pulled from all of their input directories and placed in the working directory before the config file is generated. This is explained more in-depth as the example case for [#2](https://github.com/DataBiosphere/analysis_pipeline_WDL/issues/2).
+* The twice-localized workaround is used in unique_variant_ids due to permission erros on Terra. See [#2](https://github.com/DataBiosphere/analysis_pipeline_WDL/issues/2).
```
Comment (juklucas) [inline _documentation_/for developers/cwl-vs-wdl-dev.md:35]: spelling: "errors"

**Review by juklucas — APPROVED** [review]
LGTM

File: _documentation_/for developers/symlink-workaround.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+## Reason 1: Inputs go into different folders
+_**Important note**: This describes a Cromwell execution on a local filesystem. The details of localizing files and what folders they go into changes on different backends. Some backends may not even need this workaround._
+
+The second task of the ld-prune workflow generates chromosome-level, subset GDS files in a scattered task. The third task, which is not scattered, takes in those files as inputs in order to merge them into a single GDS file.
+
+This situation, wherein a scattered task passes in inputs to a non-scattered task, passes in each instance of the scattered task's outputs into a new folder. Let's say my scattered task runs on 5 GDS files, generating 5 subset GDS files. My subsequent task is passed in those gds files like this:
+
+<img width="623" alt="Screenshot 2021-04-09 at 3 34 00 PM" src="https://user-images.githubusercontent.com/27784612/114250466-a9331f80-9952-11eb-9e09-f114f9d89e4f.png">
+
+That is to say, each GDS file now lives in its own folder within /inputs/.
```
Comment (avani-k) [inline _documentation_/for developers/symlink-workaround.md:13]: The screenshot is pretty self-explanatory but should there be a mention that the folder name is made of random numbers and cannot be predicted?

**Review by avani-k — APPROVED** [review]
I have left one comment that is more of a suggestion. Great job Ash!

File: _documentation_/for developers/symlink-workaround.md (hunk 2 above)
Comment (aofarrel) [reply _documentation_/for developers/symlink-workaround.md]: That's a good point. I added a sentence mentioning it, trying to keep it brief.

**Pull request merged**